pilot/pkg/config/kube/crd/types.go

```diff
@@ -39,7 +39,7 @@ var knownTypes = map[string]schemaType{
 		schema: model.MockConfig,
 		object: &MockConfig{
 			TypeMeta: meta_v1.TypeMeta{
-				Kind:       model.MockConfig.Type,
+				Kind:       "MockConfig",
```

Comment:
This is from a cherry-pick of a fix for an issue that was blocking me: https://github.com/istio/istio/pull/4305

Reply:
I personally don't mind - I think it is far more readable to see the actual string (which is a constant) rather than navigate to another file. Been hurt many times by a 'constant' having the wrong value or changing.